app/bundles/EmailBundle/Controller/EmailController.php

```diff
@@ -590,6 +591,15 @@ public function newAction($entity = null)
         $sections    = $model->getBuilderComponents($entity, 'sections');
         $sectionForm = $this->get('form.factory')->create('builder_section');
 
+        $pluginsAssets = [];
+        $plugins       = $this->get('mautic.helper.plugin.builder')->getBuilderPlugins();
+
+        if ($plugins) {
+            foreach ($plugins as $name => $config) {
+                $pluginsAssets[] = '<script src="/plugins/'.$name.'/Assets/js/index.js"></script>';
```

Comment:
All code related to plugin assets is not needed. Plugins have the ability to inject assets into Mautic via both the builder and Mautic itself through the CoreEvents::VIEW_INJECT_CUSTOM_ASSETS event.